src/interfaces/WF/WFArchiveFormat.ts

```diff
@@ -0,0 +1,12 @@
+type WFArchiveType = (
```

Comment:
Should the variable name be `WFArchiveFormat`?

Reply:
Fixed up all the other bits but this. Will double check in the morning when I'm on my comp!

Reply:
Great, thanks! 😎

Reply:
Fixed!